Tests/AppHealthChecksTests/AppHealthChecksTests.swift

```diff
@@ -1,12 +1,63 @@
+// FS Dependency Injection
+// Copyright (C) 2024  FREEDOM SPACE, LLC
+
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Affero General Public License as published
+//  by the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Affero General Public License for more details.
+//
+//  You should have received a copy of the GNU Affero General Public License
+//  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+//
+//  AppHealthChecksTests.swift
+//
+//
+//  Created by Mykola Buhaiov on 09.03.2023.
+//
+
+import Vapor
 import XCTest
 @testable import AppHealthChecks
 
 final class AppHealthChecksTests: XCTestCase {
-    func testExample() throws {
-        // XCTest Documentation
-        // https://developer.apple.com/documentation/xctest
+    //  swiftlint:disable implicitly_unwrapped_optional
+    var app: Application!
```

Comment:
Need resolve force unwrap

in test - let app = Application(test)
        defer { app.shutdown() }